webapp/web/configure/messages.css

```
@@ -1,3 +1,15 @@
 .standard-messages__group .accordion {
   margin-top: 0;
 }
+
+.text-edit.unsaved::before,
+.standard-message.unsaved .message::before {
+  font-family: "Font Awesome 5 Free";
+  content: "\f071";
+  display: inline-block;
+  padding-right: 3px;
+  vertical-align: middle;
+  color: var(--orange);
+  font-weight: 900;
+  margin-right: 3px;
+}
```

nit - GH was complaining about this:
```suggestion
}

```